src/main/resources/jenkins/branch/BranchSource/config.jelly

```diff
@@ -43,7 +43,8 @@
   </j:if>
   <j:set var="descriptors" value="${descriptor.buildStrategiesDescriptors(it,sourceDescriptor)}"/>
   <j:if test="${!descriptors.isEmpty()}">
-    <f:entry title="${%Build strategies}">
+    <f:entry title="${%Build strategies}" field="buildStrategies">
+    <f:entry title="${%Strategy}" field="strategyId"></f:entry>
     <f:hetero-list name="buildStrategies" items="${instance.buildStrategies}"
```

Comment:
The fake `f:entry` does not look right. You should not need to resort to tricks to have help for a standard form control. I think the problem here is that the original page used unnecessarily low-level controls. Better to use the higher-level `f:repeatableHeteroProperty` which groks databinding and should allow you to simply create a file with the conventional name, `help-buildStrategies.html`.